great_ape_safe/ape_api/cow.py

```diff
@@ -79,26 +82,25 @@ def _sell(self, sell_token, mantissa_sell, buy_token,
         fee_and_quote = self.get_fee_and_quote(sell_token, buy_token, mantissa_sell)
 
         # grab values needed to post the order to the api
-        fee_amount = int(fee_and_quote['fee']['amount'])
+        fee_amount = int(fee_and_quote["fee"]["amount"])
         if mantissa_buy:
             # overwrite quote in case order has a limit
             mantissa_buy = int(mantissa_buy)
             buy_amount_after_fee = mantissa_buy
         else:
-            buy_amount_after_fee = int(int(fee_and_quote['buyAmountAfterFee']) * coef)
+            buy_amount_after_fee = int(int(fee_and_quote["buyAmountAfterFee"]) * coef)
             pricer = interface.IOnChainPricing(
                 interface.IBribesProcessor(registry.eth.cvx_bribes_processor).pricer(),
```

Comment:
here will should use the pricer accordingly to each processor, currently it is hardcoded for cvx processor only

Reply:
fixed in 6a03250

Reply:
v cool way of leveraging `locals()` 👍